perfmetrics/scripts/gcsfuse_goofys_fio_test.py

```diff
@@ -0,0 +1,143 @@
+"""Python script to do FIO test for GCSFuse Vs GCSFuse/Goofys.
+
+To run the script:
+>> python3 gcsfuse_goofys_fio_test.py -- method1 method1_version method2 method2_version jobfile_path
+-> method1 & method can be gcsfuse or goofys.
+-> Incase of goofys specify the version as latest.
+
+The code takes input the fio test method and its version, jobfile_path as input.
+Then it will perform the fio test for each of the given method at a specific version
+and extract fio_metrics to output.txt file.
+"""
+import argparse
+import os
+from fio import fio_metrics
+
+from absl import app
+
+GCS_BUCKET = 'gke_load_test'
+GOOFYS_REPO = 'https://github.com/kahing/goofys'
+
+
+def _install_gcsfuse(version) -> None:
+  """Install gcsfuse with Specific version.
+  Args:
+    version(str): gcsfuse version to be installed.  
+  """
+  os.system(f'''curl -L -O https://github.com/GoogleCloudPlatform/gcsfuse/releases/download/v{version}/gcsfuse_{version}_amd64.deb
+            sudo dpkg --install gcsfuse_{version}_amd64.deb
+            mkdir gcs
+            gcsfuse --implicit-dirs --max-conns-per-host 100 --disable-http2 {GCS_BUCKET} gcs
```

Comment:
make all the gcs flags as constants like here: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh#L13